docs/Data/Validation/instance/Equals.md

```diff
@@ -0,0 +1,35 @@
+# `Validation#Equals<A, B>(object other)`
+
+Determine whether or not the `other` has the same value as the current `instance`.
+
+## Arguments
+
+* `other (object)`: The other object.
+
+## Types
+
+* `A`: The underlying failure type.
+* `B`: The underlying success type.
+
+## Returns
+
+* `(boolean)`: `true` for equality; otherwise, `false`.
+
+## Examples
+
+```csharp
+const failure = Validation.Failure<string, string>("a");
```

Comment:
```suggestion
var failure = Validation.Failure<string, string>("a");
```

Reply:
Updated the two of these with their explicit types. This was a mindless error from writing all the sets of documentation.